src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -488,9 +488,8 @@ public String getServiceFileName(Interface service, String packageName) {
     return getNotImplementedString("SurfaceNamer.getApiName");
   }
 
-  /** The name of the class that implements a particular proto interface. */
-  public String getFullyQualifiedApiWrapperClassName(Interface service, String packageName) {
-    return packageName + "." + getApiWrapperClassName(service);
+  public String getFullyQualifiedApiWrapperClassName(Interface interfaze, String packageName) {
+    return getNotImplementedString("SurfaceNamer.getFullyQuallifiedApiWrapperClassName");
```

Comment:
Done. 
I think [Qualified](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=qualified) is the correct spelling.